server/database/models/transactions.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const transactions = new Schema({
+  client_id: {
+    type: String,
+    required: true,
+    trim: true,
+  },
```

Comment:
`client_id` considered as Foreign key from client model, so you have to do it by this way  
```suggestion
  client_id: {
    type: Schema.Types.ObjectId,
    ref: 'client',
  },
```
projectId: {
   
  },